src/client/pages/Embark/LanguagePicker.tsx

```diff
@@ -40,13 +44,15 @@ const Divider = styled.span`
 `
 
 const Dropdown = styled.div`
+  height: 80px;
```

Comment:
Any particular reason we want to restrict the height? 🙃  

https://user-images.githubusercontent.com/6661511/112007783-3a1d9480-8b25-11eb-8073-b6fca036569c.mov

Reply:
Nope, just accidentally left it there after trying to fix styling issues. Also that `Dropdown` component of course doesn't need `display: flex` so I'll remove that as well 👍 🙏